docs/content/tutorials/nginx-dynamic-module.md

```diff
@@ -0,0 +1,118 @@
+---
+title: NGINX Ingress Controller with dynamic module
+description: |
+  Use NGINX Ingress Controller with dynamic module
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-1231"
+---
+
+## How to use an NGINX dynamic module with NGINX Ingress Controller
+
+NGINX has several dynamic modules that can be used to add additional features and capabilities. NGINX Ingress Controller can also use these same NGINX modules with NGINX Ingress Controller.
```

Comment:
```suggestion
NGINX has several dynamic modules that can be used to add additional features and capabilities. NGINX Ingress Controller can also use these same NGINX modules with NGINX Ingress Controller.
```

I would rephrase this to "additional features such as X, Y and Z": what exists right now ambi